packages/commonwealth/client/scripts/controllers/chain/cosmos/chain.utils.ts

```diff
@@ -38,5 +43,12 @@ export const getSigningClient = async (
   url: string,
   signer: OfflineSigner
 ): Promise<SigningStargateClient> => {
-  return await SigningStargateClient.connectWithSigner(url, signer);
+  const aminoTypes = new AminoTypes({
+    ...createDefaultAminoConverters(),
+    ...createAltGovAminoConverters(),
+  });
+
+  return await SigningStargateClient.connectWithSigner(url, signer, {
+    aminoTypes,
+  });
```

Comment:
This was the main fix needed, but the rest of the PR supports testing and upgrading cosmJS for this.